.github/workflows/push-image.yml

```
@@ -0,0 +1,27 @@
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          push: true
+          tags: mpppk/messagen:latest
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)